test/CheckedCRewriter/function_tests.c

```diff
@@ -0,0 +1,52 @@
+// Tests for Checked C rewriter tool.
```

Comment:
I don't think arrays of function pointer types will work yet but I'll add it in the next branch.